State transition matrix

clanghout opened the pull request:
State transition matrix wordt gemaakt en je kunt hem bekijken en saven.

clanghout committed:
change row.getCodes to return immutable set

clanghout committed:
add begin of state transition matrix controller

clanghout committed:
add extra warning check in graph create popup

clanghout committed:
Merge branch 'master' of https://github.com/clanghout/Health-Informatics-3 into state-transition-matrix

clanghout committed:
add popup for matrix creation.

The popup has default 3 values for testing reasons at the moment.
The codes can be selected by the user and will transfer to the rows of the
to be created matrix.

clanghout committed:
fix incorrect check for visualization

clanghout committed:
save changes till now

clanghout committed:
Merge branch 'master' of https://github.com/clanghout/Health-Informatics-3 into state-transition-matrix

clanghout committed:
start of tableView creation

clanghout committed:
Merge branch 'master' of https://github.com/clanghout/Health-Informatics-3 into state-transition-matrix

clanghout committed:
Merge branch 'master' of https://github.com/clanghout/Health-Informatics-3 into state-transition-matrix

Conflicts:
	src/main/java/controllers/VisualizationController.java

clanghout committed:
create tableview containing transition counts

clanghout committed:
made model local and remove dummy values

clanghout committed:
Merge branch 'graphs-bug-fix' of https://github.com/clanghout/Health-Informatics-3 into state-transition-matrix

clanghout committed:
fix matrix issues

Boudewijn26 commented:
Je code ziet er prima uit, hier en daar een wat lange methode, maar dat zei maar zo.

Als je grafieken ook werken met float values mag hij van mij.

JensLangerak commented:
moet er niet ergens een table worden gespecificeerd die gebruikt moet worden en moet men niet specificeren wat de column is die de volgorde in tijd aangeeft?

clanghout committed:
Merge branch 'master' of https://github.com/clanghout/Health-Informatics-3 into state-transition-matrix

clanghout committed:
Merge branch 'master' of https://github.com/clanghout/Health-Informatics-3 into state-transition-matrix

clanghout committed:
swap methods between abstract classes

created new abstract class so that matrix can also use shared methods

clanghout committed:
Merge branch 'master' of https://github.com/clanghout/Health-Informatics-3 into state-transition-matrix

Conflicts:
	src/main/java/controllers/PopupVisualizationController.java
	src/main/java/controllers/visualizations/BarChartController.java
	src/main/java/controllers/visualizations/BoxPlotController.java
	src/main/java/controllers/visualizations/ChartController.java

clanghout committed:
also add the new class

clanghout committed:
removed unused imports

clanghout committed:
Add functionality to the state-transition-matrix

Now a table can be specified to select codes from. After that a column can
be specified wich determines the order of the codes.

clanghout committed:
also add abstract popup controller

clanghout committed:
Merge branch 'master' of https://github.com/clanghout/Health-Informatics-3 into state-transition-matrix

Conflicts:
	src/main/java/controllers/visualizations/BarChartController.java
	src/main/java/controllers/visualizations/BoxPlotController.java

clanghout committed:
fix imports with refactor

JensLangerak commented:
ik vind hem goed

clanghout committed:
Add styleClass to matrix buttons
